package/wasm/sdks/go.go

```diff
@@ -16,8 +16,29 @@ type GoSDK struct {
 }
 
 func (s *GoSDK) Build(ctx context.Context, client *dagger.Client, hostDirectory *dagger.Directory, opts BuildOpts) error {
+	const (
+		target   = "wasm32-wasip1"
+		wasmFile = "/src/target/wasm32-wasip1/release/flipt_engine_wasm.wasm"
+	)
+
+	rust := client.Container().From("rust:1.83.0-bookworm").
+		WithWorkdir("/src").
+		WithDirectory("/src/flipt-engine-ffi", hostDirectory.Directory("flipt-engine-ffi")).
+		WithDirectory("/src/flipt-engine-wasm", hostDirectory.Directory("flipt-engine-wasm"), dagger.ContainerWithDirectoryOpts{
+			Exclude: []string{"pkg/", ".gitignore"},
+		}).
+		WithDirectory("/src/flipt-engine-wasm-js", hostDirectory.Directory("flipt-engine-wasm-js"), dagger.ContainerWithDirectoryOpts{
+			Exclude: []string{"pkg/", ".gitignore"},
+		}).
+		WithDirectory("/src/flipt-evaluation", hostDirectory.Directory("flipt-evaluation")).
+		WithFile("/src/Cargo.toml", hostDirectory.File("Cargo.toml")).
```

Comment:
should this be a `baseContainer` method in BaseSDK?

Reply:
I think I'll do that if there is another (non-JS) wasm based SDK